src/common/types/chains.types.ts

```diff
@@ -0,0 +1,9 @@
+
```

Comment:
I'm not against the idea (and I did originally put this file in there) but, on reflection I moved it here because the majority of our types are in the common directory at the moment. If you put a ticket in the backlog to move these over to the types directory we can handle this in a single PR.

Reply:
OK, if you ask me, my suggestion is like so:
- for new types, we can use `src/types` directory.
- for original types, we can use the current structure for now and handle them via main refactoring related tickets.

I will follow you.